### ISSUE TITLE
Helper arrows remain when side menu closed

### ISSUE THREAD

**Issue by domenic-d** [LaserWeb/LaserWeb4#254]
![screen shot 2017-04-09 at 12 34 27 pm](https://cloud.githubusercontent.com/assets/27034222/24834173/26d87046-1d21-11e7-91de-87b54812f563.png)
![screen shot 2017-04-09 at 12 34 40 pm](https://cloud.githubusercontent.com/assets/27034222/24834175/29061d96-1d21-11e7-864c-1f656a0207ff.png)

**Comment by jorgerobles** [comment]
Cannot consider it a bug. What's the scenario?

**Comment by domenic-d** [comment]
@jorgerobles I'm not exactly sure what you mean by a scenario, but here it goes.
What does happen:
Step 1: The user minimises the CAM tab.
Step 2: The red helper arrow titled "Click here to begin' now points to the COMMS menu.

What should happen:
Step 1: The user minimises the CAM tab.
Step 2: The red helper arrow titled "Click here to begin' should be hidden along with the CAM tab.

If you cannot consider it a bug, then perhaps you may consider it 'undesirable' or 'misleading' behaviour?

**Comment by jorgerobles** [comment]
ok, let me rephrase :)
 In which case the advice of  there's no documents to work is not desirable.? 
 My scenario is a noob user that runs laserweb and clicks on the tabs and then the panel is hidden. Has no loaded files, I find desirable to hint him where to start.

I'm not discarding the issue, just trying to empathize the situation :)

**Comment by domenic-d** [comment]
It must at least point to the correct tab otherwise we are confusing the 'noob' even more. Please look carefully at my images and explanation.

**Comment by cojarbi** [comment]
Maybe have a noob default option so it shows the help and an option to disable it?

**Comment by jorgerobles** [comment]
@domenic-d You are right. Sorry, misread your former comment.

**Comment by jorgerobles** [comment]
Patched using same notification scheme than settings. If full balloon is desirable, @tbfleming please explain the way to do with getbounds :)
